Sabiur.rahman/aws integration excluded regions api (#6945)
DataDog/documentation: content/en/api/integrations_aws/aws_create.md

    An object (in the form `{"namespace1":true/false, "namespace2":true/false}`) that enables or disables metric collection for specific AWS namespaces for this AWS account only. A list of namespaces can be found at the `/v1/integration/aws/available_namespace_rules` endpoint.

* **`excluded_regions`** [*optional*, *default*=**None**]:

>     Array of AWS regions that will be excluded from metrics collection. 

Reviewer: ```suggestion
    An array of AWS regions to exclude from metrics collection. 
```
